buildpacks/builder.go

```diff
@@ -52,7 +57,7 @@ type Impl interface {
 
 // NewBuilder instantiates a Buildpack-based Builder
 func NewBuilder(options ...Option) *Builder {
-	b := &Builder{}
+	b := &Builder{name: DefaultName}
```

Comment:
Could this just use the constant and eliminate the `var` above?
```suggestion
	b := &Builder{name: builders.Pack}
```

Reply:
Yes, that of course works well, but this way has some benefits for godoc.  The exported "DefaultName" is highlighted quite prominently in godocs, whereas if I directly reference the variable the auto-entry in the docs is skipped.  Also, it's been